makefiles/vars.inc.mk

```
@@ -19,6 +19,7 @@ export MCU                   # The MCU, set by the board's Makefile.include, or
 export INCLUDES              # The extra include paths, set by the various Makefile.include files.
 export CXXINCLUDES           # The extra include paths for c++, set by the various Makefile.include files.
 export NATIVEINCLUDES        # The native include paths, set by the various native Makefile.include files.
+export DEVELHELP             # Set to 1 to enable code in RIOT that does safety checking which is not needed in a production environment but helps in the development process.
```

```suggestion
export DEVELHELP             # Set to 1 to enable code in RIOT that does safety checking which might not be needed in a production environment but helps in the development process.
```

The message is wrong though, `enabling` MPI stackguard is something you would like to do in production as well I would think...

---

I guess we don't need to worry about this if we get the targets identified...

---

Never a bad thing to document a variable though, but out of scope if you target-export in deed